templates/scenarios/ts/spfx-tab/.vscode/launch.json

```diff
@@ -86,6 +86,86 @@
                 "hidden": true
             },
             "internalConsoleOptions": "neverOpen"
+        },
+        {
+            "name": "Attach to Frontend in Outlook (Edge)",
+            "type": "pwa-msedge",
+            "request": "launch",
+            "url": "https://outlook.office.com/host/${local:teamsAppInternalId}?${account-hint}",
+            "webRoot": "${workspaceRoot}/SPFx",
```

Comment:
The `SPFx` folder has been renamed to `src` in v3

Reply:
Thanks the reminder! Updated new and old ones as well.